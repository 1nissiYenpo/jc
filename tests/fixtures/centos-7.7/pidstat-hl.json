[{"time":1646859134,"uid":0,"pid":1,"percent_usr":0.0,"percent_system":0.03,"percent_guest":0.0,"percent_cpu":0.03,"cpu":0,"command":"/usr/lib/systemd/systemd --switched-root --system --deserialize 22"},{"time":1646859134,"uid":0,"pid":6,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"ksoftirqd/0"},{"time":1646859134,"uid":0,"pid":9,"percent_usr":0.0,"percent_system":0.01,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"rcu_sched"},{"time":1646859134,"uid":0,"pid":11,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"watchdog/0"},{"time":1646859134,"uid":0,"pid":32,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"khugepaged"},{"time":1646859134,"uid":0,"pid":308,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"scsi_eh_1"},{"time":1646859134,"uid":0,"pid":309,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"kworker/u256:2"},{"time":1646859134,"uid":0,"pid":319,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"scsi_eh_2"},{"time":1646859134,"uid":0,"pid":356,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"kworker/u256:4"},{"time":1646859134,"uid":0,"pid":357,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"irq/16-vmwgfx"},{"time":1646859134,"uid":0,"pid":465,"percent_usr":0.0,"percent_system":0.01,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"xfsaild/dm-0"},{"time":1646859134,"uid":0,"pid":466,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"kworker/0:1H"},{"time":1646859134,"uid":0,"pid":543,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/lib/systemd/systemd-journald"},{"time":1646859134,"uid":0,"pid":564,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/sbin/lvmetad -f"},{"time":1646859134,"uid":0,"pid":577,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/lib/systemd/systemd-udevd"},{"time":1646859134,"uid":0,"pid":752,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/sbin/auditd"},{"time":1646859134,"uid":0,"pid":779,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/libexec/bluetooth/bluetoothd"},{"time":1646859134,"uid":999,"pid":780,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/lib/polkit-1/polkitd --no-debug"},{"time":1646859134,"uid":0,"pid":782,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/sbin/smartd -n -q never"},{"time":1646859134,"uid":0,"pid":784,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/lib/systemd/systemd-logind"},{"time":1646859134,"uid":81,"pid":787,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation"},{"time":1646859134,"uid":998,"pid":790,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/sbin/chronyd"},{"time":1646859134,"uid":0,"pid":834,"percent_usr":0.0,"percent_system":0.01,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"/usr/sbin/crond -n"},{"time":1646859134,"uid":0,"pid":847,"percent_usr":0.01,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"/usr/bin/python2 -Es /usr/sbin/firewalld --nofork --nopid"},{"time":1646859134,"uid":0,"pid":849,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/sbin/agetty --keep-baud 115200,38400,9600 ttyS0 vt220"},{"time":1646859134,"uid":0,"pid":852,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"login -- kbrazil"},{"time":1646859134,"uid":0,"pid":882,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/sbin/NetworkManager --no-daemon"},{"time":1646859134,"uid":0,"pid":1031,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/sbin/dhclient -d -q -sf /usr/libexec/nm-dhcp-helper -pf /var/run/dhclient-ens33.pid -lf /var/lib/NetworkManager/dhclient-d92ec"},{"time":1646859134,"uid":0,"pid":1220,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/sbin/sshd -D"},{"time":1646859134,"uid":0,"pid":1221,"percent_usr":0.06,"percent_system":0.03,"percent_guest":0.0,"percent_cpu":0.09,"cpu":0,"command":"/usr/bin/dockerd-current --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current --default-runtime=docker-runc --exec"},{"time":1646859134,"uid":0,"pid":1222,"percent_usr":0.01,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"/usr/bin/python2 -Es /usr/sbin/tuned -l -P"},{"time":1646859134,"uid":0,"pid":1225,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"/usr/sbin/rsyslogd -n"},{"time":1646859134,"uid":0,"pid":1293,"percent_usr":0.04,"percent_system":0.02,"percent_guest":0.0,"percent_cpu":0.05,"cpu":0,"command":"/usr/bin/docker-containerd-current -l unix:///var/run/docker/libcontainerd/docker-containerd.sock --metrics-interval=0 --start-"},{"time":1646859134,"uid":0,"pid":1496,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"/usr/libexec/postfix/master -w"},{"time":1646859134,"uid":89,"pid":1512,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"qmgr -l -t unix -u"},{"time":1646859134,"uid":1000,"pid":1852,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"-bash"},{"time":1646859134,"uid":0,"pid":1872,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"sshd: kbrazil [priv]"},{"time":1646859134,"uid":1000,"pid":1876,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"sshd: kbrazil@pts/0"},{"time":1646859134,"uid":1000,"pid":1877,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"-bash"},{"time":1646859134,"uid":0,"pid":2232,"percent_usr":0.0,"percent_system":0.01,"percent_guest":0.0,"percent_cpu":0.01,"cpu":0,"command":"kworker/0:1"},{"time":1646859134,"uid":0,"pid":2239,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"kworker/0:3"},{"time":1646859134,"uid":89,"pid":2240,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"pickup -l -t unix -u"},{"time":1646859134,"uid":0,"pid":2263,"percent_usr":0.0,"percent_system":0.0,"percent_guest":0.0,"percent_cpu":0.0,"cpu":0,"command":"kworker/0:0"}]
