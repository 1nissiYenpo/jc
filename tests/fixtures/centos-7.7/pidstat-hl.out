Linux 3.10.0-1062.1.2.el7.x86_64 (localhost) 	03/09/2022 	_x86_64_	(1 CPU)

#      Time   UID       PID    %usr %system  %guest    %CPU   CPU  Command
 1646859134     0         1    0.00    0.03    0.00    0.03     0  /usr/lib/systemd/systemd --switched-root --system --deserialize 22 
 1646859134     0         6    0.00    0.00    0.00    0.00     0  ksoftirqd/0
 1646859134     0         9    0.00    0.01    0.00    0.01     0  rcu_sched
 1646859134     0        11    0.00    0.00    0.00    0.00     0  watchdog/0
 1646859134     0        32    0.00    0.00    0.00    0.00     0  khugepaged
 1646859134     0       308    0.00    0.00    0.00    0.00     0  scsi_eh_1
 1646859134     0       309    0.00    0.00    0.00    0.00     0  kworker/u256:2
 1646859134     0       319    0.00    0.00    0.00    0.00     0  scsi_eh_2
 1646859134     0       356    0.00    0.00    0.00    0.00     0  kworker/u256:4
 1646859134     0       357    0.00    0.00    0.00    0.00     0  irq/16-vmwgfx
 1646859134     0       465    0.00    0.01    0.00    0.01     0  xfsaild/dm-0
 1646859134     0       466    0.00    0.00    0.00    0.00     0  kworker/0:1H
 1646859134     0       543    0.00    0.00    0.00    0.00     0  /usr/lib/systemd/systemd-journald 
 1646859134     0       564    0.00    0.00    0.00    0.00     0  /usr/sbin/lvmetad -f 
 1646859134     0       577    0.00    0.00    0.00    0.00     0  /usr/lib/systemd/systemd-udevd 
 1646859134     0       752    0.00    0.00    0.00    0.00     0  /sbin/auditd 
 1646859134     0       779    0.00    0.00    0.00    0.00     0  /usr/libexec/bluetooth/bluetoothd 
 1646859134   999       780    0.00    0.00    0.00    0.00     0  /usr/lib/polkit-1/polkitd --no-debug 
 1646859134     0       782    0.00    0.00    0.00    0.00     0  /usr/sbin/smartd -n -q never 
 1646859134     0       784    0.00    0.00    0.00    0.00     0  /usr/lib/systemd/systemd-logind 
 1646859134    81       787    0.00    0.00    0.00    0.00     0  /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation 
 1646859134   998       790    0.00    0.00    0.00    0.00     0  /usr/sbin/chronyd 
 1646859134     0       834    0.00    0.01    0.00    0.01     0  /usr/sbin/crond -n 
 1646859134     0       847    0.01    0.00    0.00    0.01     0  /usr/bin/python2 -Es /usr/sbin/firewalld --nofork --nopid 
 1646859134     0       849    0.00    0.00    0.00    0.00     0  /sbin/agetty --keep-baud 115200,38400,9600 ttyS0 vt220 
 1646859134     0       852    0.00    0.00    0.00    0.00     0  login -- kbrazil      
 1646859134     0       882    0.00    0.00    0.00    0.00     0  /usr/sbin/NetworkManager --no-daemon 
 1646859134     0      1031    0.00    0.00    0.00    0.00     0  /sbin/dhclient -d -q -sf /usr/libexec/nm-dhcp-helper -pf /var/run/dhclient-ens33.pid -lf /var/lib/NetworkManager/dhclient-d92ec
 1646859134     0      1220    0.00    0.00    0.00    0.00     0  /usr/sbin/sshd -D 
 1646859134     0      1221    0.06    0.03    0.00    0.09     0  /usr/bin/dockerd-current --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current --default-runtime=docker-runc --exec
 1646859134     0      1222    0.01    0.00    0.00    0.01     0  /usr/bin/python2 -Es /usr/sbin/tuned -l -P 
 1646859134     0      1225    0.00    0.00    0.00    0.01     0  /usr/sbin/rsyslogd -n 
 1646859134     0      1293    0.04    0.02    0.00    0.05     0  /usr/bin/docker-containerd-current -l unix:///var/run/docker/libcontainerd/docker-containerd.sock --metrics-interval=0 --start-
 1646859134     0      1496    0.00    0.00    0.00    0.00     0  /usr/libexec/postfix/master -w 
 1646859134    89      1512    0.00    0.00    0.00    0.00     0  qmgr -l -t unix -u 
 1646859134  1000      1852    0.00    0.00    0.00    0.00     0  -bash 
 1646859134     0      1872    0.00    0.00    0.00    0.00     0  sshd: kbrazil [priv] 
 1646859134  1000      1876    0.00    0.00    0.00    0.00     0  sshd: kbrazil@pts/0  
 1646859134  1000      1877    0.00    0.00    0.00    0.00     0  -bash 
 1646859134     0      2232    0.00    0.01    0.00    0.01     0  kworker/0:1
 1646859134     0      2239    0.00    0.00    0.00    0.00     0  kworker/0:3
 1646859134    89      2240    0.00    0.00    0.00    0.00     0  pickup -l -t unix -u 
 1646859134     0      2263    0.00    0.00    0.00    0.00     0  kworker/0:0
